pkg/server/api/api.go

```diff
@@ -65,23 +64,3 @@ func DoCNI(url string, req interface{}, socketPath string) ([]byte, error) {
 
 	return body, nil
 }
-
-// GetAPIEndpoint returns endpoint URL for multus-daemon
-func GetAPIEndpoint(endpoint string) string {
-	return fmt.Sprintf("http://dummy%s", endpoint)
-}
-
-// CreateDelegateRequest creates Request for delegate API request
-func CreateDelegateRequest(cniCommand, cniContainerID, cniNetNS, cniIFName, podNamespace, podName, podUID string, cniConfig []byte, interfaceAttributes *DelegateInterfaceAttributes) *Request {
-	return &Request{
-		Env: map[string]string{
-			"CNI_COMMAND":     strings.ToUpper(cniCommand),
-			"CNI_CONTAINERID": cniContainerID,
-			"CNI_NETNS":       cniNetNS,
-			"CNI_IFNAME":      cniIFName,
-			"CNI_ARGS":        fmt.Sprintf("K8S_POD_NAMESPACE=%s;K8S_POD_NAME=%s;K8S_POD_UID=%s", podNamespace, podName, podUID),
-		},
-		Config:              cniConfig,
-		InterfaceAttributes: interfaceAttributes,
-	}
-}
```

Comment:
I don't run golang-lint, however, I guess that it is shown as 'unused code' in lint, however these function is API and consumed by other repository, so please keep these code (including 'string' import).